src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -614,11 +598,7 @@ const formConfig = {
           showPagePerItem: true,
           keepInPageOnReview: false,
           title: item => `${applicantWording(item)} address selection`,
-          // Only show if we have addresses to pull from:
-          depends: (formData, index) =>
-            (index && index > 0) || // We will have app0's address
-            (get('street', formData?.certifierAddress) ||
-              get('street', formData?.sponsorAddress)),
+          depends: (formData, index) => page15aDepends(formData, index),
```

Comment:
Pulled this logic into a separate function so it wouldn't be as cluttered/would be easier to test.